### PR TITLE
Update Bash 5.0 with patches 12..16

### PR DESCRIPTION
**Pull request by Mno-hime**
**Changes**: mostly related to problems with history invocation

### CODE REVIEW

File: components/shell/bash/Makefile [hunk 1]
```diff
@@ -124,16 +122,12 @@ COMPONENT_TEST_MASTER		= \
 # avoid permission issues for other users running these tests
 COMPONENT_POST_TEST_ACTION =	rm -f /tmp/xx
 
-build: $(BUILD_64)
-
 install: $(INSTALL_64)
```
Comment (pyhalov) [inline components/shell/bash/Makefile]: Move it to COMPONENT_POST_INSTALL_ACTION ?

Comment (Mno-hime) [reply components/shell/bash/Makefile]: Changed `COMPONENT_POST_TEST_ACTION` to `COMPONENT_POST_INSTALL_ACTION`.

Comment (pyhalov) [reply components/shell/bash/Makefile]: Wait, I haven't meant it.
I've meant that lines `(cd $(PROTOUSRSHAREDIR)/locale ; cp -Rp "en@boldquot" en ;...` could be moved to COMPONENT_POST_INSTALL_ACTION, and so we don't need `install: $(INSTALL_64)...` rule

**Comment by alarcher** [comment]
@Mno-hime Any issue blocking this one?

**Pull request merged**